package.json

```diff
@@ -1,16 +1,19 @@
 {
   "name": "postmate",
-  "version": "1.3.0",
+  "version": "1.4.0",
   "description": "A powerful, simple, promise-based postMessage library",
   "main": "build/postmate.min.js",
   "files": [
     "build",
     "src"
   ],
   "devDependencies": {
-    "babel-eslint": "^8.2.1",
-    "babel-plugin-transform-class-properties": "^6.11.5",
-    "babel-preset-es2015": "^6.24.1",
+    "@babel/cli": "^7.0.0-beta.40",
+    "@babel/core": "^7.0.0-beta.40",
+    "@babel/plugin-proposal-class-properties": "^7.0.0-beta.40",
+    "@babel/preset-env": "^7.0.0-beta.40",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^22.2.2",
```

Comment:
Do we need all of these dependencies? `@babel/core`, `babel-core`, `@babel/cli`, etc

Reply:
And you're sure we need to install a command line interface as a dependency?

Reply:
I believe so! Lemme check.